02-microarray/clustering/clustering_example.Rmd

```diff
@@ -1,20 +1,111 @@
+ 
+***
+
+# How to run this example
+
+For general information about our tutorials, and for the basic software packages you will need; see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.nb.html#how-this-tutorial-is-structured).
+We recommend taking a look at our [Resources for Learning R](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.nb.html#resources-for-learning-r) if you have not written code in R before. 
+
+## About the dataset we are using for this example
+
+For this example analysis, we will use this [zebrafish microarray dataset](
+https://www.refine.bio/experiments/GSE12955/gene-expression-profile-of-zebrafish-kidney-side-population-sp-cells).
+This dataset has two experimental groups of lymphoid cells from zebrafish kidney.
+Using Hoechst fluorescence intensity, these were subdivided into the SP and MP populations. 
+
+## Obtaining the dataset from refine.bio
+
+For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.nb.html#how-to-get-the-data). 
+
+Go to this [dataset's page on refine.bio](https://www.refine.bio/experiments/GSE12955/gene-expression-profile-of-zebrafish-kidney-side-population-sp-cells).  
+
+Click the "Download Now" button on the right side of this screen.  
+
+<img src="screenshots/download-now.png" width=200>  
+
+Fill out the pop up window with your email and our Terms and Conditions:  
+
+<img src="screenshots/download-email.png" width=500>  
+
+It make take a few minutes for the dataset to process.
+You will get an email when it is ready. 
+Follow the prompt to download a zip file that has a name with a series of letters and numbers and ends in `.zip`. 
+Double clicking should unzip this for you and create a folder of the same name.  
+<img src="screenshots/download-folder-structure.png" width=400> 
+
+For more details on the contents of this folder see [these docs on refine.bio](http://docs.refine.bio/en/latest/main_text.html#rna-seq-sample-compendium-download-folder).  
+
+The `<experiment_accession_id>` folder has the data and metadata TSV files you will need for this example analysis.
+Experiment accession ids usually look something like `GSE1235` or `SRP12345`. 
+
+### Obtaining this example R notebook
+
+To run the examples yourself, you need to [download the Rmd by clicking this link](https://alexslemonade.github.io/refinebio-examples/02-microarray/clustering/clustering_example.Rmd). 
+Clicking this link will most likely send this to your downloads folder on your computer. 
+
+If you are unfamiliar with R notebooks, see our [introduction to using Rmds](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.nb.html#how-to-get-and-use-rmds) for more resources and explanations. 
```

Comment:
The notebook download should come **first** in my opinion and should draw one's eye at the very beginning of the notebook.

Imagining a workflow like:

1. Get and open the notebook locally.
2. The user is instructed to put the notebook somewhere they want to work with it.
3. Okay now we run our first chunks! And these chunks create the directories you need to use this. You may have to change these to work with your own data.
4. Now that we're set up, let's download some files from refine.bio.
5. While you're waiting, let me tell you a little bit about the dataset (a teaser if you will).
6. When you have the download locally, you need to move it.
7. Here's what it should look like for you to know it's right.
  * Maybe this looks like: a fenced code block that shows the correct output of `list.files()` then you explain each directory and file followed by a chunk that runs `list.files()` (_Is yours the same dear reader?_)

The motivation here, in part, is _show, don't tell_. And this seems easy enough to blow past if you're browsing the web version where you're not creating the folders, etc. You'd then say _more_ about the dataset when you get to the business of using the dataset.

Side note: you could consider some HTML tricks to make the section about setting up locally collapsible (https://www.w3schools.com/tags/tag_details.asp) 

@dvenprasad can we get your professional opinion(s)?

Reply:
The current flow is very much inline with how people go about analysis. i.e they get the data first and then think about how to run the analysis but it doesn't feel flexible like they can skip ahead if they are more confident.

The proposed flow, however does provide an opportunity for users to decide to skip getting refine.bio data and run it with their own depending on their confidence with R. Maybe some folks are here just for the notebook. ( i don't know if we're presenting a way for them only the notebooks elsewhere, currently my assumption is we're not doing that)

I would probably say 'hey you can put your data here in this folder structure we've created or you can run it with the example dataset we're using' before step 4. 

Also would move step 5 before step 4 just because I feel that they might not return to this page as soon as they start the download, i.e they might linger on refine.bio.

Reply:
>The proposed flow, however does provide an opportunity for users to decide to skip getting refine.bio data and run it with their own depending on their confidence with R. Maybe some folks are here just for the notebook. ( i don't know if we're presenting a way for them only the notebooks elsewhere, currently my assumption is we're not doing that)

I've reorganized the workflow (and put it in a template format) mostly according to Jackie's outline above. I think as far as the concerns here, I set up the headers and the TOC in such a way that someone can skip right to the "meat" of the analysis if that is what they are actually here for. 

As an alternative to either thing you guys have mentioned, I put a the `# Using your own data with this analysis` section at the very end of the analysis. But it part of the bigger option in the TOC, so someone could skip right to it if they are confident, but less experienced individuals don't have to be interrupted by the "next steps" until they've gotten through the original dataset.